### ISSUE TITLE
Is there a bifaciality factor included in the current logic ? 

### ISSUE THREAD

**Issue by UGuntupalli** [pvcaptest/pvcaptest#58]
Since a lot of the current market deployments are bifacial modules, does the current method account for bifaciality anywhere ? 
If yes, which method ? 
If no, can we start looking at adding it ?

**Comment by bt-** [comment]
@UGuntupalli,

Yes, the current version of the package can be used to test bifacial projects, but there is not any bifacial specific functionality yet. 

The user can change the regression equation `CapData.regression_formula` and regress against whatever columns of data in the `CapData.data_filtered` attribute desired, which allows a lot of flexibility including the ability to run tests on bifacial systems.

If you are following the adjustment for a bifacial project proposed by the [CCR/NREL paper](https://www.nrel.gov/docs/fy20osti/73982.pdf), then you just need to assign a value to the `CapData.regression_cols` `poa` key that points to a column containing E_Total. This approach requires calculating E_Total from E_POA, E_Rear, and the bifaciality factor manually and adding it to the `CapData.data_filtered` dataframe. 

It would be helpful to add method(s) to facilitate this process. I haven't thought through how that should work fully. My initial thought is that you could make the `poa` value of `CapData.regression_cols` a `CapData.column_group` that is a group of E_Rear and E_poa and then have a method to calculate E_Total that takes the bifaciality as an argument. The values of `CapData.regression_cols` can be actual column names of the `CapData.data_filtered` dataframe or they can be keys of the `CapData.column_groups`. This functionality exists to be able to point to a group of measurements of the same type, e.g. irradiance from multiple POA pyranometers, and aggregate those into a single value to regress against. So, this approach to calculating E_Total would be similar.

There is more to discuss about this approach. One of the parts of pvcaptest that is due for a major refactoring is the column grouping. I have not opened an issue on that, but I have been working/thinking about new approaches.
